.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: ruby
+
+rvm:
```

Comment:
I'm okay with this, but I'd like to see jRuby and 2.0 since they are more relevant to us (nothing wrong with putting our thumb on that scale)